tools/pipelines/build-docs.yml

```diff
@@ -26,12 +35,20 @@ variables:
     value: ${{ eq(variables['Build.SourceBranchName'], variables['RELEASE_BRANCH']) }}
   - name: validRun
     value: ${{ or(variables.releasePipeline, variables.n1Pipeline, variables.latestPipeline) }}
```

Comment:
(not really related) are the `releasePipeline` or `n1Pipeline` variables still available? looks like they were defined in some of the ymls that got deleted, but i'm not sure if they're being provided elsewhere

Reply:
The variables are still defined in ADO, so I left these in place for now.